extensions/amp-story/1.0/amp-story.js

```diff
@@ -1490,16 +1494,28 @@ export class AmpStory extends AMP.BaseElement {
    * @private
    */
   polyfillDvh_(size) {
-    const {height, width} = size;
-    if (height === 0 && width === 0) {
+    const {height} = size;
+    if (height === 0) {
       return;
     }
-    this.storeService_.dispatch(Action.SET_PAGE_SIZE, {height, width});
     setImportantStyles(this.win.document.documentElement, {
       '--story-dvh': px(height / 100),
     });
   }
 
+  /**
+   * Handles resize events and sets the store service's width and height.
+   * @private
+   */
+  onViewportResize_() {
+    const page = this.element.querySelector(`amp-story-page[active]`);
+    const layoutBox = page?./*OK*/ getLayoutBox();
```

Comment:
There's a couple of checks here if the page is null, and the width and height are also not null. When could the active page be null, if we're always running this after the `active` attr is set?

Note: You can use `devAssert()` to make sure an element is not null, otherwise it throws an error.